### DISCUSSION TITLE
windows  text-generation-webui方式部署失败

### DISCUSSION THREAD

**Discussion by qianxifu** [ymcui/Chinese-LLaMA-Alpaca-2#339] [Q&A]
D:\pro\aigc\text-generation-webui>python server.py --model chinese-alpaca-2-7b-hf --chat
Traceback (most recent call last):
  File "D:\pro\aigc\text-generation-webui\server.py", line 14, in <module>
    import gradio as gr
ModuleNotFoundError: No module named 'gradio'

gradio包一直安装不成功，试了很多次了。

D:\pro\aigc\text-generation-webui>pip install gradio
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1122)'))': /simple/gradio/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1122)'))': /simple/gradio/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1122)'))': /simple/gradio/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1122)'))': /simple/gradio/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1122)'))': /simple/gradio/
Could not fetch URL https://pypi.org/simple/gradio/: There was a problem confirming the ssl certificate: HTTPSConnectionPool(host='pypi.org', port=443): Max retries exceeded with url: /simple/gradio/ (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1122)'))) - skipping
ERROR: Could not find a version that satisfies the requirement gradio (from versions: none)
ERROR: No matching distribution found for gradio

**Comment by ymcui** [comment]
去gradio那边的issue里搜索一下试试。
